charts/bookkeeper-operator/templates/bookkeeper.pravega.io_bookkeeperclusters_crd.yaml

```diff
@@ -2825,6 +2825,10 @@ spec:
                       type: string
                   type: object
                 type: array
+              upgradeTimeout:
+                description: This is used to schedule the timeout value for rollback/upgrade
```

Comment:
Suggestion --> `This is used to schedule the timeout value in minutes for rollback/upgrade`